engine/lite/kafka/api/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/api/LiteKafkaSource.scala

```diff
@@ -9,4 +10,10 @@ trait LiteKafkaSource extends BaseLiteSource[ConsumerRecord[Array[Byte], Array[B
 
   def topics: List[String]
 
+  def transform(record: ConsumerRecord[Array[Byte], Array[Byte]]): Context = {
```

Comment:
what about `LiteKafkaSourceImpl`? I wonder - should it be the single implementation? This addition should be more of a fallback

Reply:
you are right, that's why some other tests didn't pass, I added it 
but if I delete the method here i think i need to add it in TestComponentProvider so maybe it should stay here as well?